cypress/integration/ButtonStrip/Accepts_children.feature

```diff
@@ -0,0 +1,5 @@
+Feature: The ButtonStrip renders children
```

Comment:
What the ButtonStrip does, is provide some spacing to its children. So I'd like to see a test that asserts that. Having said that, I know this component is due to be removed and replaced by some generic spacing component. So you can probably keep this one as is.

Reply:
Yeah here I've also avoided testing the styles (see earlier comment). Would be nice to test things like that though, for sure.